src/order/boolean_algebra.lean

```diff
@@ -1,11 +1,13 @@
 /-
 Copyright (c) 2017 Johannes Hölzl. All rights reserved.
+Copyright (c) 2020 Chris Upshaw. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
-Authors: Johannes Hölzl
+Authors: Johannes Hölzl, Chris Upshaw
```

Comment:
The purpose of the authors list is generally to identify the individuals who had a significant impact on the design of the Lean code in that file. See the discussion at the end of [this file](https://leanprover-community.github.io/contribute/style.html#header-and-imports).

Similarly, we generally only include the creator of the file in the list of copyrights.